website/content/en/docs/releases/experimental.md

```diff
@@ -19,6 +19,8 @@ The following commands are experimental and subject to change:
 - `limactl snapshot *`
 - `limactl tunnel`
 - `limactl template *`
+- `limactl plugins` plugin mechanism (see [CLI plugins](../config/plugin/cli))
+  - `limactl-*`
```

Comment:
This line should be in the previous block ("The following features are experimental and subject to change:") under the "External drivers:" line, and look like this:

```suggestion
- `limactl plugins:` plugin mechanism (see [CLI plugins](../config/plugin/cli))
```

There are no specific commands to list down here because plugins would be invoked just like a builtin command, and we can't list `limactl *` here. 😄